packages-exp/auth-exp/package.json

```diff
@@ -39,6 +39,7 @@
     "@firebase/app-types-exp": "0.x",
     "@firebase/auth-types-exp": "0.x",
     "@firebase/auth-exp": "0.x",
+    "@firebase/component": "0.x",
```

Comment:
These shouldn't be a peerDepenencies except for `app-exp` and `app-types-exp`. And they should use exact versions .

Reply:
Added component but I think https://github.com/firebase/firebase-js-sdk/pull/3654 does the rest